test/mocks/config/mocks.h

```diff
@@ -59,6 +60,9 @@ class MockSubscriptionFactory : public SubscriptionFactory {
   MOCK_METHOD0(messageValidationVisitor, ProtobufMessage::ValidationVisitor&());
 
   MockSubscription* subscription_{};
+
+  // map from the config hash to the callbacks.
+  // std::unordered_map<uint64_t, SubscriptionCallbacks*> callbacks_map_;
```

Comment:
@htuch found this change from your recent refactoring... 

this is the part I found the mocks deviates from the implementation, only one callbacks_ can be reference here. Not sure if it's worth to use a map with map[hash(config_source)] to each xDS callbacks instance...

I kinda of get my needs covered in unit test though.

Reply:
Probably not worth adding unless it provides some substantive improvement to the tests.

Reply:
got it. removed these comments.